pridėtas aktyvių užduočių spausdinimas

sigitasmickis opened the pull request:
taip pat pataisyta baigtų užduočių sąlyga, bei įvestas spaudinimo kategorijos atspindėjimas print metode

Sigitas-Mickis committed:
pridėtas aktyvių užduočių spausdinimas

taip pat pataisyta baigtų užduočių sąlyga, bei įvestas spaudinimo kategorijos atspindėjimas print metode

ca-java commented on src/main/TaskManager.java:
```diff
@@ -44,19 +46,24 @@ public boolean update(Task oldTask, Task newTask) {
         List<Task> result = new ArrayList<>();
         for (Task task: tasks) {
-            if (task.isCompleted())
+            if (!task.isCompleted())
```
`!task.isCompleted()` sudės į list'ą visus tasks, kurie nėra pabaigti, o reikia visų, kurie jau yra baigti

ca-java commented on src/main/TaskManager.java:
```diff
@@ -44,19 +46,24 @@ public boolean update(Task oldTask, Task newTask) {
-        return null;
+        // return incomplete tasks
+        List<Task> result = new ArrayList<>();
+        for (Task task: tasks) {
```
Čia priešingai, reikia grąžinti visus, kurie dar nėra completed (ty vis dar active). 

Taip pat, rekomenduočiau `Task` klasėje pakeisti kintamojo `activeStatus` name į `completed` (default reikšmė keičiasi į false), kad neklaidintų iš išorės ką reiškia status.

sigitasmickis commented:
Sveiki, nesupratau komentaro.... Pagal mano atsiųstą kodą, gaunu taip:

---- All tasks readout ----
name: Call Brother, description:
name: Order Pizza 🍕, description:
---- End of All tasks readout ----

---- All tasks readout ----
name: Order Pizza 🍕, description:
name: Call Mother, description:
---- End of All tasks readout ----

---- Completed tasks readout ----
name: Call Mother, description:
---- End of Completed tasks readout ----

---- Active tasks readout ----
name: Order Pizza 🍕, description:
---- End of Active tasks readout ----

Būtent, active status programoje yra false, jeigu užduotis baigta, ką ir padariau. Bet ok, pakeisiu tą požymį į completed, bus mažiau painiavos.

From: ca-java ***@***.***>
Sent: Sunday, November 27, 2022 5:33 PM
To: ca-java/todo ***@***.***>
Cc: Sigitas Mickis ***@***.***>; Author ***@***.***>
Subject: Re: [ca-java/todo] pridėtas aktyvių užduočių spausdinimas (PR #7)


@ca-java commented on this pull request.

________________________________

In src/main/TaskManager.java<https://github.com/ca-java/todo/pull/7#discussion_r1032950318>:

>          // return completed tasks



         List<Task> result = new ArrayList<>();



         for (Task task: tasks) {



-            if (task.isCompleted())



+            if (!task.isCompleted())



!task.isCompleted() sudės į list'ą visus tasks, kurie nėra pabaigti, o reikia visų, kurie jau yra baigti

________________________________

In src/main/TaskManager.java<https://github.com/ca-java/todo/pull/7#discussion_r1032951106>:

>                  result.add(task);



         }



         return result;



     }







     public List<Task> getActiveTasks(){



-        // return completed tasks



-        return null;



+        // return incomplete tasks



+        List<Task> result = new ArrayList<>();



+        for (Task task: tasks) {



Čia priešingai, reikia grąžinti visus, kurie dar nėra completed (ty vis dar active).

Taip pat, rekomenduočiau Task klasėje pakeisti kintamojo activeStatus name į completed (default reikšmė keičiasi į false), kad neklaidintų iš išorės ką reiškia status.

—
Reply to this email directly, view it on GitHub<https://github.com/ca-java/todo/pull/7#pullrequestreview-1195002342>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AI7WQS6QH2OXX4H3OS4WNDDWKN5KXANCNFSM6AAAAAASLJQK6M>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

Sigitas-Mickis committed:
Pakeistas užduoties baigimo požymis į completionStatus

pagal naują baigimo požymį suderinti užbaigų ir aktyvių užduočių išspausdinimo metodai

Sigitas-Mickis committed:
suderinti metodai pagal naują baigimo požymį completionStatus

Sigitas-Mickis committed:
Merge branch 'active-tasks-sm' of https://github.com/ca-java/todo into active-tasks-sm
